guppylang/definition/custom.py

```diff
@@ -131,28 +131,6 @@ def parse(self, globals: "Globals", sources: SourceMap) -> "CustomFunctionDef":
             sig is not None,
         )
 
-    def compile_call(
-        self,
-        args: list[Wire],
-        type_args: Inst,
-        dfg: DFContainer,
-        ctx: CompilerContext,
-        node: AstNode,
-        function_ty: ht.FunctionType,
-    ) -> Sequence[Wire]:
```

Comment:
drive-by: This function is actually unused. We should never call `compile_call` on a `RawCustomFunctionDef`, only on `CustomFunctionDef`s